ee/localserver/krypto-ec-middleware.go

```diff
@@ -27,16 +27,17 @@ type v2CmdRequestType struct {
 }
 
 type kryptoEcMiddleware struct {
-	signer       crypto.Signer
-	counterParty ecdsa.PublicKey
-	logger       log.Logger
+	localDbSigner, hardwareSigner crypto.Signer
+	counterParty                  ecdsa.PublicKey
+	logger                        log.Logger
 }
 
-func newKryptoEcMiddleware(logger log.Logger, signer crypto.Signer, counteParty ecdsa.PublicKey) *kryptoEcMiddleware {
+func newKryptoEcMiddleware(logger log.Logger, localDbSigner, hardwareSigner crypto.Signer, counteParty ecdsa.PublicKey) *kryptoEcMiddleware {
 	return &kryptoEcMiddleware{
-		signer:       signer,
-		counterParty: counteParty,
-		logger:       log.With(logger, "keytype", "ec"),
+		localDbSigner:  localDbSigner,
+		hardwareSigner: hardwareSigner,
+		counterParty:   counteParty,
```

Comment:
Not your code, but `counteParty` typo

Reply:
it is my code, just part of this pr =), fixed